tests/test_columns.py

```diff
@@ -73,6 +77,10 @@ async def test_model_crud():
     product = await Product.objects.get(pk=product.pk)
     assert product.created.year == datetime.datetime.now().year
     assert product.created_day == datetime.date.today()
+    assert product.created_date.today() == datetime.date.today()
```

Comment:
Maybe something like this?
```suggestion
    assert product.created_date == datetime.date.today()
```